composer.json

```
@@ -3,7 +3,7 @@
     "type": "neos-package",
     "name": "psmb/splitadd",
     "require": {
-        "neos/neos": "^8.0 || ^7.0 || ^5.0 || ^4.0 || ^3.3"
+        "neos/neos-ui": "~7.3 || ^8.0"
```

```suggestion
        "neos/neos-ui": "~8.3"
```

hmm sorry i changed my mind ^^ lets go with 8.3 as i just recently discussed that 7.3 is half as dead already (no bugfix support anymore... in a few days i guess ^^)

Also this will make it easier to maintain this plugin as only ckeditor 16 has to be supported and 12 has not be kept in mind.

---

If it works I don't see the need to fix what's not broken and raise the constraint.
Upgrading CKE might be breaking for more plugins, so I don't expect it for Neos 8 versions (or are there other plans @mhsdesign ?).

With Neos 9 I can imagine other issues with the node creation part of the plugin, so I would expect the plugin to be updated (and version constrained) for newer versions anyway.

---

I meant that we did upgrade the ckeditor in between 7.3 and 8.3 so it would probably be saner to just support 8.3 :D

And i think i must withdraw my approval slightly. Sometimes i have the feeling that the splitting doesnt work on 8.3 :(
I have to debug this yet. ^^

---

Yes okay then lets stick with it ;)

I actually got the plugin to work again but found thousand other bugs: https://github.com/psmb/Psmb.SplitAdd/issues/9